Fix typo in Market.__init__

mnwhite opened the pull request:
Delinting at PyCon created a typo in the argument list for Market, now fixed.  This addresses the same issue as just-closed PR from CDC #285

pkofod commented:
This is sort of why I suggested at some point (and didn't do it myself) that it can be a good idea to simply gather all the model types in a test set and just `init` them all. Just to check that stuff still can be initialized after a change. It's a crude check, but it is quite important that all models can be initiated after all :)

llorracc commented:
Release notes: Bug fix

mnwhite commented:
Adjustment: No release notes needed.  This was a one character typo
introduced at PyCon and corrected a few days later.  There's no need to
mention this in the release notes.

On Thu, May 16, 2019 at 8:53 AM Christopher Llorracc Carroll <
notifications@github.com> wrote:

> Release notes: Bug fix
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/econ-ark/HARK/pull/286?email_source=notifications&email_token=ADKRAFLGMJARLNCL253FDQLPVVKNVA5CNFSM4HMRQNV2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODVRWY7Q#issuecomment-493055102>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADKRAFI6BMYGYLZDL7RAJE3PVVKNVANCNFSM4HMRQNVQ>
> .
>
